css/index.css

```diff
@@ -104,14 +147,44 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
```

Comment:
Good work only overriding what you need to in the child boxes and letting the .box class take care of the rest of the styling.